src/composables/queries/usePoolsQuery.ts

```diff
@@ -61,8 +67,28 @@ export default function usePoolsQuery(
       await store.dispatch('registry/injectTokens', uninjectedTokens);
     }
 
+    const poolsWithLiquidityMining = pools.map(pool => {
```

Comment:
I would suggest that this should actually be part of the serialisation of the subgraph request here: https://github.com/balancer-labs/frontend-v2/blob/develop/src/services/balancer/subgraph/entities/pools/index.ts#L47-L64

So that liquidityMiningAPY is just an additional attribute of the decorated Pool type.